src/config/datasets.yml

```diff
@@ -295,7 +295,7 @@ collections:
 
   landsat-c2-l2:
     category: Imagery
-    headerImg: ./images/landsat-c2-l2-hero.png
+    headerImg: ./images/landsat-hero.png
```

Comment:
Looks like this file is actually `*.jpg`. 

With this change, the landsat group hero is now the same as c2-l2 hero - do you think we should preserve the c2-l2 hero to differentiate the pages?

Reply:
Yep, that is a jpg. Will push the correction.

>With this change, the landsat group hero is now the same as c2-l2 hero - do you think we should preserve the c2-l2 hero to differentiate the pages?

If I understand you correctly, I think that will work. To be clear, what I'm trying to avoid is a group hero that does not "match" (in a color sense) any of the collection thumbs on that same page (see image of current group page). It is improved now that the c2-l1 thumb has been updated, but it still feels a bit "clashy" to me, particularly the greens in the hero and in the c2-l2 thumb. 

<img width="1580" alt="image" src="https://user-images.githubusercontent.com/7013017/161623334-1646d9e7-96f8-4453-846b-e4a7f0e585fe.png">

But, yeah, we can preserve the c2-l2 hero, which would differentiate things. It would not "match" its thumb on the group page, but maybe that doesn't matter?

But, really, I'm fine with whatever you think is good, so let me know which way you want to go.

Reply:
For continuity and differentiation, if I can get past the colors in the above screenshot, maybe it's best to use the 8-c2-l2 thumb for the group thumb, the 8-c2-l2 hero for the group hero, and the c2-l1 and c2-l2 thumbs on the group page as shown above. And have the c2-l1 and c2-l2 heroes on their dedicated pages "match" their thumbs. If that makes sense (which it may not, this is rather confusing to put into text), I can make the necessary updates.

Reply:
Ha, that is a bit confusing to try and sort through as text, but yes I think I agree with that. Net result is we're not reusing the group thumb/header on any of the collection pages, and each collection has a consistently styled header and thumb. I do feel that clash on the group, but I think it's probably helpful to have an association where the clicking them thumb will result in a page with a similar header image on it. So +1 on that proposal.

Reply:
Sounds good, will update and ping you when complete.

Reply:
OK, I think I've got things put together as discussed above.